docs/pages/installation/helm.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Installing Teleport using Helm
+description: How to install Teleport using Helm.
+---
+
+Teleport maintains Helm charts for services and plugins, including self-hosted
+Auth Service and Proxy service deployments, Teleport Agents, the Event Handler,
+and Access Request plugins.
+
+## Installing Teleport as a Cluster
+
+The [teleport-cluster](../reference/helm-reference/teleport-cluster.mdx) chart deploys the teleport daemon on Kubernetes with preset configurations for the Auth and Proxy Services and support for any Teleport service configuration.
```

Comment:
```suggestion
The [teleport-cluster](../reference/helm-reference/teleport-cluster.mdx) chart deploys the `teleport` daemon on Kubernetes with preset configurations for the Auth and Proxy Services and support for any Teleport service configuration.
```
Since we're talking about the binary and not Teleport the product.